deepomatic/api/http_helper.py

```diff
@@ -79,7 +79,9 @@ def __init__(self, app_id=None, api_key=None, verify_ssl=None,
             raise DeepomaticException("Please specify 'api_key' either by passing it to the client"
                                       " or by defining the DEEPOMATIC_API_KEY environment variable.")
 
-        if not isinstance(version, string_types):
+        if version is None:
```

Comment:
I think we might want to handle empty string as well so that it does not prefix with a `v`.